packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -141,7 +257,7 @@ describe('EDiscovery Content API Tests', () => {
       };
       const result = webex.internal.ediscovery.getContentSummaryBySpaceId(uuid, uuid)
         .then((res) => {
-          expect(res).to.equal(mockCachedSpace);
+          expect(res.body).to.equal(mockCachedSpace);
```

Comment:
getContentSummaryBySpaceId now returns a response type result even if it hits the cache